selfdrive/controls/plannerd.py

```diff
@@ -22,6 +22,7 @@ def publish_ui_plan(sm, pm, lateral_planner, longitudinal_planner):
   uiPlan.position.x = np.interp(plan_odo, model_odo, lateral_planner.lat_mpc.x_sol[:,0]).tolist()
   uiPlan.position.y = np.interp(plan_odo, model_odo, lateral_planner.lat_mpc.x_sol[:,1]).tolist()
   uiPlan.position.z = np.interp(plan_odo, model_odo, lateral_planner.path_xyz[:,2]).tolist()
+  uiPlan.accel = longitudinal_planner.a_desired_trajectory_full.tolist()
```

Comment:
let's split the uiPlan and planner changes into their own PR

Reply:
already asked @HaraldSchafer to merge this separately